### PR TITLE
feat: Initialize script

### PR DESCRIPTION
**Pull request by fussel178**
### Summary 

Semi-automated initialization via GitHub Actions :tropical_fish: 

### Details 

Add an initialization helper in form of a GitHub Action which sets up the needed steps in the `README.md` automatically.
It can be triggered via a `workflow_dispatch` event.
Simply go to the _Actions_ Tab in the GitHub UI and choose the `Initialize` Action.
Then click `Run workflow` and enter your preferences like so:

Screenshot_20210427_091359

This action creates a file named `.github/initialized` to prevent unintended re-initialization of the project some time later.

### Additional information 

You can find my test repository here:
https://github.com/fussel178/telestion-project-automated-setup

### Related links 

- [x] I have signed the individual contributor's license agreement **and sent** it to the board of the WüSpace e. V. organization.

### CODE REVIEW

File: application/build.gradle [hunk 1]
```diff
@@ -4,14 +4,14 @@ plugins {
...
 
-group 'de.wuespace.telestion.project.template'
+group '##GROUP_ID##'
 version '1.0-SNAPSHOT'
 
 sourceCompatibility = gradle.sourceCompatibility
 targetCompatibility = gradle.targetCompatibility
 
 application {
-    mainClassName = 'de.wuespace.telestion.Main'
```
Comment (jvpichowski) [inline application/build.gradle:14]: The main class is always the same. It is located in a library and not in the project specific application (PSA). At the moment it is called Telestion, not Main. I am sorry I forgot that yesterday.

File: README.md [hunk 2]
```diff
@@ -4,13 +4,20 @@ This is a template for Telestion applications.
 It helps you to set up the publishing of Telestion applications.
 Dependencies to the [main repo](https://github.com/wuespace/telestion-core) are included, too.
 
-## Necessary Changes
+## Initialization
+
+Please go to the _Actions_ Tab in the GitHub UI and choose the `Initialize` Action.
+Then click `Run workflow` and enter your preferences like so:
+
+![Screenshot_20210426_221738](https://user-images.githubusercontent.com/52416718/116158314-f991d780-a6dd-11eb-9961-0a80ce9b7950.png)
```
Comment (jvpichowski) [inline README.md:12]: You might want to change this image when the main class is removed.

File: application/build.gradle [hunk 3]
```diff
@@ -4,14 +4,14 @@ plugins {
     id 'application'
 }
 
-group 'de.wuespace.telestion.project.template'
+group '##GROUP_ID##'
```
Comment (jvpichowski) [inline application/build.gradle:7]: If you change the group id you might also change the given folder structure in `src/java/main`. Is this possible?

**Review by jvpichowski — CHANGES_REQUESTED** [review]
I really like that contribution. Will we see a subsequent PR that initializes the psc?

File: application/build.gradle (hunk 1 above)
Comment (fussel178) [reply application/build.gradle:14]: Ah! Good point!
So is the current main class name `de.wuespace.telestion.Telestion`?

Comment (jvpichowski) [reply application/build.gradle:14]: Yes it is

File: application/build.gradle (hunk 3 above)
Comment (fussel178) [reply application/build.gradle:7]: Yes, that's possible!
I currently thinking, if it's better to first create the folder structure and generate from it the group name or the other way around. :thinking:

File: application/build.gradle (hunk 1 above)
Comment (jvpichowski) [reply application/build.gradle:14]: Or better yes it will be when we have moved the core repo to wuespace 😄

File: application/build.gradle (hunk 3 above)
Comment (jvpichowski) [reply application/build.gradle:7]: I don't know 😄 I think the order doesn't matter.

File: application/build.gradle (hunk 1 above)
Comment (fussel178) [reply application/build.gradle:14]: Fixed in https://github.com/wuespace/telestion-project-template/pull/4/commits/60c2458e5b0f67e77e2dd9e84359cc41d3b35536

File: README.md (hunk 2 above)
Comment (fussel178) [reply README.md]: Updated in https://github.com/wuespace/telestion-project-template/pull/4/commits/de3af067fff69bd9f4b63a6f45199ff72dc70345

File: application/build.gradle (hunk 3 above)
Comment (fussel178) [reply application/build.gradle:7]: Feature added in https://github.com/wuespace/telestion-project-template/pull/4/commits/3f223b18db93fde76b0206e7fd7ca12cc7f9ed0f.
Please take a look at the changes.

Example can be found here:
https://github.com/fussel178/telestion-project-automated-setup/tree/test-7/application/src/main/java

Comment (jvpichowski) [reply application/build.gradle:7]: Perfect

**Review by jvpichowski — APPROVED** [review]

**Review by pklaschka — APPROVED** [review]
Overall, it looks good. It doesn't have to be within this Pull Request, but we should probably, in addition, add instructions for manual (non-GitHub-Actions) steps for the setup, as we cannot expect everyone to use GitHub. Other than that, it looks great 🙂

**Pull request merged**